Dockerfile

```diff
@@ -42,6 +42,10 @@ EXPOSE 42222:22
 EXPOSE 8042:8042
 EXPOSE 60010:60010
 
+# For Spark
+EXPOSE 8080:8080
```

Comment:
Why is port 8080 exposed? Spark External web UI uses port 18080 - is this a typo forgetting a "1"?